### PR TITLE
Initial PR

### PR DESCRIPTION
**Pull request by smittytone**

### CODE REVIEW

**Comment by smittytone** [comment]
@ppetrosh, this shouldn't need much of a review. It's a quick library for a display unit which I'm including in an article on the Grove System for the Dev Center. So it just needs a brief look, any changes doing, and then posting.

File: grove.tm1637.device.lib.nut [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+const GROVE_TM1637_HIGH = 1;
+const GROVE_TM1637_LOW = 0;
+const GROVE_TM1637_ADDR_AUTO = 0x40;
+const GROVE_TM1637_ADDR_FIXED = 0x44;
+const GROVE_TM1637_DATA_CMD = 0xC0;
+const GROVE_TM1637_DISPLAY_CMD = 0x88;
+
+class GroveTM1637 {
+
+    _dataPin = null;
```
Comment (ppetrosh) [inline grove.tm1637.device.lib.nut]: Does it need to have a `static VERSION`?

**Review by ppetrosh — APPROVED** [review]

File: grove.tm1637.device.lib.nut (hunk 1 above)
Comment (smittytone) [reply grove.tm1637.device.lib.nut]: Good point - totally forgot about that. Will add it in.

**Comment by ppetrosh** [comment]
@smittytone do we need this to be published as a library?

**Comment by smittytone** [comment]
@ppetrosh, we can, but it's not essential. This is more for sample code than for something customers will use in the field, so I'd say no. I plan to update the code to remove the #requires from the examples and read me - this can be a cut'n'paste library.

**Comment by ppetrosh** [comment]
I see thank you @smittytone

**Pull request merged**